main.py

```diff
@@ -18,5 +18,45 @@
 # This step should be performed before importing any of the
 # third party libraries.
 
+import logging
+import sys
+import time
+
 import vm_config
 vm_config.configure()
+
+# pylint: disable=wrong-import-position
+from core.services import job_services
+import vmconf
+
+def main():
+    """Main process of VM."""
+    try:
+        job_data = job_services.get_next_job()
+        if job_data is None:
+            logging.info('No pending job requests.')
```

Comment:
Actually I have kept the format of logging same as GAE's log formats which includes all necessary details, I guess.